testnode-scripts/config.ts

```diff
@@ -248,6 +252,24 @@ function writeConfigs(argv: any) {
     posterConfig.node["seq-coordinator"].enable = true
     posterConfig.node["batch-poster"].enable = true
     fs.writeFileSync(path.join(consts.configpath, "poster_config.json"), JSON.stringify(posterConfig))
+
+    let validationNodeConfig = JSON.parse(JSON.stringify({
+        "persistent": {
+	        "chain": "local"
+        },
+        "ws": {
+            "addr": "0.0.0.0",
+			"api": "validation",
+        },
+        "http": {
+            "addr": "",
+        },
+		"validation": {
+			"api-auth": false,
```

Comment:
The "proper" way to connect to validation node should be with authenticated api. I want test node to use that. You can create a jwt during initialization and use it.